vspace/src/main/webapp/WEB-INF/views/staff/modules/slides/slide.jsp

```diff
@@ -628,6 +648,27 @@ $(window).on('load', function () {
 		</div>
 	</div>
 </div>
+
+<div id="loginAlert" class="modal" tabindex="-1" role="dialog"
+	backdrop="static"
+	style="display: none; background-color: rgba(0, 0, 0, 0.5);">
+	<div class="modal-dialog">
+		<div class="modal-content">
+			<div class="modal-header">
+				<h5 class="modal-title">Login</h5>
+			</div>
+			<div class="modal-body">
+				<h6>
+					You are not logged In, Please login.
```

Comment:
`In` shouldn't be capatilized and either turn `,` into a `.` or lowercase `Please`.